Misc/NEWS.d/next/Library/2022-07-14-10-28-10.gh-issue-94844.NJ3W80.rst

```diff
@@ -0,0 +1,2 @@
+:meth:`shutil.make_archive` now accepts any :class:`os.PathLike`. Patch by
```

Comment:
It was undocumented behavior, and we do not yet guarantee it in future versions, so no NEWS entry is needed.

Reply:
Done; could you add skip-news label please?